pkgs/development/python-modules/pystache/default.nix

```
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "pystache";
-  version = "0.6.5";
+  version = "0.6.6";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and set `build-system = [ setuptools ]`